src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -0,0 +1,83 @@
+module Webui::Staging::WorkflowHelper
+  def build_progress(staging_project)
+    total = 0
+    final = 0
+
+    staging_project.building_repositories.each do |r|
+      total += r[:tobuild] + r[:final]
+      final += r[:final]
+    end
+
+    return 100 if total == 0
+    # if we have building repositories, make sure we don't exceed 99
+    [final * 100 / total, 99].min
+  end
+
+  def review_progress(staging_project)
+    staged_requests_numbers = staging_project.staged_requests.pluck(:number)
+    total = Review.where(bs_request: staging_project.staged_requests).count
+    missing = staging_project.missing_reviews.count { |missing_review| staged_requests_numbers.include?(missing_review[:request]) }
```

Comment:
same here!!!! :see_no_evil: 

and why the `()` around `missing_review[:request]`? :thinking:

Reply:
The parentheses are for the `.include?` method. Somehow, it's rendered on another line on GitHub.